app/models/iiif_canvas_indexer_enqueuer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+##
+# For a given object in an exhibit, gets the correct manifest, and enqueues
+# canvas/page indexing jobs from that manifest.
+class IiifCanvasIndexerEnqueuer
+  attr_reader :exhibit, :druid
+
+  def initialize(exhibit, druid)
+    @exhibit = exhibit
+    @druid = druid
+  end
+
+  def enqueue_jobs
+    canvases.each do |canvas|
+      canvas.other_content.each do |other_content|
+        # Don't bother to index unless there is an annotationList content to do so.
+        next unless other_content['@type'] == 'sc:AnnotationList'
```

Comment:
What would you think about making this string a constant at the top of the class?  I have gotten in the habit of trying to keep values like this out of the body of methods...not sure I've ultimately seen the benefit of doing it, but it has seemed cleaner.  Totally up to you!

Reply:
👍

Reply:
Did you want to use the constant in the body of this method?